### PR TITLE
Add support for previousDataSetItem to ConversionDataSource

### PR DESCRIPTION
**Pull request by chances**
[RICE-129](https://apptree.myjetbrains.com/youtrack/issue/RICE-129): Migrate WO Translation to FAMISCloud Connector

### CODE REVIEW

File: app/sdk/converter/ObjectConverter.java [hunk 1]
```diff
@@ -135,7 +135,7 @@ private static ParserContext getParserContext() {
             attributeMeta = inferMetaData(index, fieldClass);
         }
         if (!isFieldClassSupportedForType(fieldClass, attributeMeta.getAttributeType())) {
-            throw new UnsupportedAttributeException(fieldClass, attributeMeta.getAttributeType());
+//            throw new UnsupportedAttributeException(fieldClass, attributeMeta.getAttributeType());
         }
```
Comment (alexisandreason) [inline app/sdk/converter/ObjectConverter.java]: I don't think this error should be ignored.

Comment (SamOrozco) [reply app/sdk/converter/ObjectConverter.java]: I don't think it should be either

Comment (chances) [reply app/sdk/converter/ObjectConverter.java]: Whoops, you're right. That was accidentally commited

File: app/sdk/controllers/ConversionController.java [hunk 2]
```diff
@@ -48,20 +58,21 @@ public Result getConversionConfiguration(String endpointName) {
...
-
-    protected DataSetItem dataSetItemFromRequest(ServiceConfiguration configuration, Http.Request request) {
+    protected DataSetItem dataSetItemFromRequest(ServiceConfiguration configuration, Http.Request request, String fieldName) {
         ObjectNode node = (ObjectNode) request.body().asJson();
         DataSet newDataSet = new DataSet(configuration.getAttributes());
-        ObjectNode sourceItem = (ObjectNode) node.findValue("dataSetItem");
-        return dataSetItemForJSON(sourceItem, newDataSet, false, new HashMap<>());
+        JsonNode sourceItemNode = node.findValue(fieldName);
+        if (sourceItemNode == null || sourceItemNode instanceof NullNode) {
+            return null;
```
Comment (SamOrozco) [inline app/sdk/controllers/ConversionController.java]: I'm not a big fan of returning null, should you throw an exception instead?

**Pull request merged**